### PR TITLE
Create isTrue function and make it available to templates.

### PR DESCRIPTION
**Pull request by cedriessen**
We have a bunch of boolean environment variables (flags) where a resilient function to check for truth would be helpful.

### CODE REVIEW

**Comment by cedriessen** [comment]
This commit also makes lower and uppercase functions available as `lower` and `upper`.

File: template.go [hunk 1]
```diff
@@ -68,6 +68,14 @@ func add(arg1, arg2 int) int {
 	return arg1 + arg2
 }
 
+func isTrue(bool string) bool {
+  b, err := strconv.ParseBool(strings.ToLower(bool))
```
Comment (jwilder) [inline template.go]: `strings.ToLower` is not needed because `ParseBool` already handles 't' and 'true'.

File: template.go [hunk 2]
```diff
@@ -68,6 +68,14 @@ func add(arg1, arg2 int) int {
 	return arg1 + arg2
 }
 
+func isTrue(bool string) bool {
```
Comment (jwilder) [inline template.go]: Can you change `bool string` to `s string`?

**Comment by jwilder** [comment]
Can you run `go fmt` on the files you've changed.  The formatting looks off.

File: template.go (hunk 1 above)
Comment (cedriessen) [inline template.go]: I wanted the function to return true regardless of the spelling, i.e. "TRUE", "tRuE", etc. should also return true. Does that make sense?

File: template.go (hunk 2 above)
Comment (cedriessen) [inline template.go]: Done.

**Comment by cedriessen** [comment]
Formatted the code.

**Comment by myniva** [comment]
What is the status on this? The changes look quite nice would help me solve some current issues.

**Comment by jwilder** [comment]
Thanks @cedriessen!

**Pull request merged**